keyboards/ergodox_ez/keymaps/bpruitt-goddard/keymap.c

```diff
@@ -0,0 +1,275 @@
+// An Ergodox EZ keymap mostly following the programmer's dvorak layout.
+// There is a standard QWERTY layer as well
+//
+// See the README.md file for an image of this keymap.
+
+#include QMK_KEYBOARD_H
+
+// The layers that we are defining for this keyboards.
+#define BASE 0
+#define NUM 1
+#define QWERTY 2
+#define MAC 3
+
+// The Tap Dance identifiers, used in the TD keycode and tap_dance_actions array.
+#define TAP_MACRO 0
+
+// SAFE_RANGE must be used to tag the first element of the enum.
+// DYNAMIC_MACRO_RANGE must always be the last element of the enum if other
+// values are added (as its value is used to create a couple of other keycodes
+// after it).
+enum custom_keycodes {
+    MC_ARROW = SAFE_RANGE,
+    DYNAMIC_MACRO_RANGE
+};
+
+// A 'transparent' key code (that falls back to the layers below it).
+#define ___ KC_TRANSPARENT
+
+// A 'blocking' key code. Does nothing but prevent falling back to another layer.
+#define XXX KC_NO
+
+// Some combined keys (one normal keycode when tapped and one modifier or layer
+// toggle when held).
+#define SPC_RALT  MT(MOD_RALT, KC_SPC)  // SPACE key and right alt modifier.
+
+// The most portable copy/paste keys (windows (mostly), linux, and some terminal emulators).
+#define MK_CUT    LSFT(KC_DEL)  // shift + delete
+#define MK_COPY   LCTL(KC_INS)  // ctrl + insert
+#define MK_PASTE  LSFT(KC_INS)  // shift + insert
+//Move mac desktop spaces
+#define MAC_L     LGUI(LSFT(KC_UP)) // cmd + shift + up
+#define MAC_R     LGUI(LSFT(KC_DOWN)) // cmd + shift + down
+
+// This file must be included after DYNAMIC_MACRO_RANGE is defined...
+#include "dynamic_macro.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  // Layer 0: basic keys.
+  [BASE] = KEYMAP(
```

Comment:
Could you change the `KEYMAP`s to `LAYOUT_ergodox`?

Reply:
I hadn't seen that! I updated my `KEYMAP`s to be the new pretty format (see current PR version) ✨.